codesign/lib/src/google_cloud_storage.dart

```diff
@@ -10,19 +10,10 @@ import 'package:process/process.dart';
 import './utils.dart';
 
 /// Utility function class to handle upload/download of files from/to google cloud.
```

Comment:
This is a bit more than a utility. It's a service for interacting with the local `gsutil`

Reply:
Sure, updated description.